package.json

```diff
@@ -40,7 +41,7 @@
     "rollup": "^0.50.0",
     "rollup-plugin-babel": "^3.0.2",
     "sierra-library": "^2.0.0",
-    "webpack": ">= 2.2.0",
+    "webpack": ">=2.0 <4.0.0",
```

Comment:
please move it into "hot module reload" PR as the current version supports webpack 4

Reply:
"html-webpack-plugin" is not dependency of sass-webpack-plugin

Reply:
I just change webpack version from `devDependencies`, because of the demo need `html-webpack-plugin`.

Reply:
ahh, sorry missed that

Reply:
you can clone https://github.com/AepKill/sass-webpack-plugin.git to try it, it's work ok.